components/event-service/internal/events/subscribed/events.go

```diff
@@ -0,0 +1,126 @@
+package subscribed
+
+import (
+	subscriptions "github.com/kyma-project/kyma/components/event-bus/generated/push/clientset/versioned"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+//EventsClient interface
+type EventsClient interface {
+	GetSubscribedEvents(appName string) (Events, error)
+}
+
+//Events represents collection of all events with subscriptions
+type Events struct {
```

Comment:
Consider changing the implementation to `type Events []Event` then you can avoid duplication.